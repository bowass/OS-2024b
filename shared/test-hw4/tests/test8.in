23219
60800
54520
55010
58558
5921
13059
57972
59012
47295
10213
33599
34499
34496
45004
9001
47798
24711
4375
51254
30225
57010
27034
26953
54705
39989
38244
53513
30831
11233
63628
3328
16081
45913
65208
62800
3884
25088
4022
23994
61786
19629
340
56835
30529
35421
60586
61942
51805
37307
51675
42889
4522
18098
29453
50601
41410
5765
44269
31772
19614
47810
27537
54
27163
1116
24502
43446
20656
37795
27105
8464
36662
20196
25424
55854
52511
32850
1395
9151
29529
60218
34229
58605
56185
17461
38531
7208
57901
32578
38503
45993
63134
42511
30583
27915
61079
12790
35514
38805
13084
62712
46610
21460
41659
3650
31292
33515
54447
63704
7517
45228
9962
55457
30459
42145
20666
32665
51536
3464
65398
17640
24550
29038
53534
40216
55950
31129
65216
1025
2066
7448
33158
10666
26126
39664
9690
22222
56962
39238
17160
40605
64471
12814
32017
46852
24720
10067
42123
28815
12774
22796
59342
31567
12538
55998
19246
51130
26637
50182
32916
42375
51178
58548
53216
36292
8697
35332
53490
21122
23579
43696
28172
11941
37120
26533
17080
45363
41245
22551
33812
26205
37585
17100
13254
7960
9835
36737
30312
8338
48254
31925
25224
65420
37150
2902
56455
45373
14207
26628
49260
20540
5773
10174
34812
13385
51138
4119
42495
61799
48009
42692
37522
22711
55945
37491
20098
37261
57343
63304
29748
21250
6363
17950
44596
10201
20869
28602
2363
49854
3286
45689
20076
33972
18272
12155
37231
6230
61913
3056
54112
37402
30997
19486
39918
35325
1946
43598
38576
53794
35223
4797
53304
51344
15562
35269
1948
33569
43532
48440
54041
8589
12289
37940
40239
20893
32530
9340
31760
27231
49635
20238
62102
59361
4500
44585
34790
14568
65533
16228
2911
28971
9115
14124
40378
33837
63804
61143
7949
21571
38237
34842
62455
32630
7462
12313
48799
3589
11345
30423
1724
30414
14088
37585
16688
26499
45345
49642
5667
31549
49002
50213
17022
47187
22458
15634
2937
38261
1677
13709
9850
59973
9709
65066
14321
45515
40539
51321
24271
16586
1515
45842
35860
2777
33929
2730
49847
60189
3690
57945
17593
53121
45931
22286
43815
56357
4902
35720
3608
1623
5410
24768
39276
56133
45270
28524
33479
40266
42970
20307
37380
37575
3261
7061
61331
58575
24540
13678
7514
37803
474
18424
31203
34282
48653
33652
31362
8396
53853
36144
40181
6563
46955
3190
63429
43902
17077
32287
47452
9096
16419
16642
14913
14025
59322
46836
38333
53489
58695
46559
43942
25515
26642
2657
1366
42776
11137
38636
12242
40340
9120
35684
4679
45877
36820
28564
49308
10996
27755
22947
53631
1378
24954
53255
4886
25955
45677
7017
59994
39697
50311
62878
13172
3029
54655
30551
894
62970
9476
18292
39207
60095
54265
36385
11268
38633
53801
61112
18432
36966
1506
52877
11643
34963
15806
56874
4357
41851
24349
4737
52113
36027
26057
60796
501
57391
63946
63885
54760
6248
52205
765
12709
59880
12763
26349
23698
56625
19147
4477
58705
50021
21218
26976
30394
21437
20532
26165
55977
52385
18541
56204
56530
3489
43347
889
14885
11186
23775
51889