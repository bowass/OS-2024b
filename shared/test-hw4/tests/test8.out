Virtual address: 23219 Physical address: 179 Value: -84
Virtual address: 60800 Physical address: 384 Value: 0
Virtual address: 54520 Physical address: 760 Value: 0
Virtual address: 55010 Physical address: 994 Value: 53
Virtual address: 58558 Physical address: 1214 Value: 57
Virtual address: 5921 Physical address: 1313 Value: 0
Virtual address: 13059 Physical address: 1539 Value: -64
Virtual address: 57972 Physical address: 1908 Value: 0
Virtual address: 59012 Physical address: 2180 Value: 0
Virtual address: 47295 Physical address: 2495 Value: 47
Virtual address: 10213 Physical address: 2789 Value: 0
Virtual address: 33599 Physical address: 2879 Value: -49
Virtual address: 34499 Physical address: 3267 Value: -80
Virtual address: 34496 Physical address: 3264 Value: 0
Virtual address: 45004 Physical address: 3532 Value: 0
Virtual address: 9001 Physical address: 3625 Value: 0
Virtual address: 47798 Physical address: 4022 Value: 46
Virtual address: 24711 Physical address: 4231 Value: 33
Virtual address: 4375 Physical address: 4375 Value: 69
Virtual address: 51254 Physical address: 4662 Value: 50
Virtual address: 30225 Physical address: 4881 Value: 0
Virtual address: 57010 Physical address: 5298 Value: 55
Virtual address: 27034 Physical address: 5530 Value: 26
Virtual address: 26953 Physical address: 5449 Value: 0
Virtual address: 54705 Physical address: 5809 Value: 0
Virtual address: 39989 Physical address: 5941 Value: 0
Virtual address: 38244 Physical address: 6244 Value: 0
Virtual address: 53513 Physical address: 6409 Value: 0
Virtual address: 30831 Physical address: 6767 Value: 27
Virtual address: 11233 Physical address: 7137 Value: 0
Virtual address: 63628 Physical address: 7308 Value: 0
Virtual address: 3328 Physical address: 7424 Value: 0
Virtual address: 16081 Physical address: 7889 Value: 0
Virtual address: 45913 Physical address: 8025 Value: 0
Virtual address: 65208 Physical address: 8376 Value: 0
Virtual address: 62800 Physical address: 8528 Value: 0
Virtual address: 3884 Physical address: 8748 Value: 0
Virtual address: 25088 Physical address: 8960 Value: 0
Virtual address: 4022 Physical address: 8886 Value: 3
Virtual address: 23994 Physical address: 9402 Value: 23
Virtual address: 61786 Physical address: 9562 Value: 60
Virtual address: 19629 Physical address: 9901 Value: 0
Virtual address: 340 Physical address: 10068 Value: 0
Virtual address: 56835 Physical address: 5123 Value: -128
Virtual address: 30529 Physical address: 10305 Value: 0
Virtual address: 35421 Physical address: 10589 Value: 0
Virtual address: 60586 Physical address: 10922 Value: 59
Virtual address: 61942 Physical address: 9718 Value: 60
Virtual address: 51805 Physical address: 11101 Value: 0
Virtual address: 37307 Physical address: 11451 Value: 110
Virtual address: 51675 Physical address: 11739 Value: 118
Virtual address: 42889 Physical address: 11913 Value: 0
Virtual address: 4522 Physical address: 4522 Value: 4
Virtual address: 18098 Physical address: 12210 Value: 17
Virtual address: 29453 Physical address: 12301 Value: 0
Virtual address: 50601 Physical address: 12713 Value: 0
Virtual address: 41410 Physical address: 12994 Value: 40
Virtual address: 5765 Physical address: 13189 Value: 0
Virtual address: 44269 Physical address: 13549 Value: 0
Virtual address: 31772 Physical address: 13596 Value: 0
Virtual address: 19614 Physical address: 9886 Value: 19
Virtual address: 47810 Physical address: 4034 Value: 46
Virtual address: 27537 Physical address: 13969 Value: 0
Virtual address: 54 Physical address: 14134 Value: 0
Virtual address: 27163 Physical address: 14363 Value: -122
Virtual address: 1116 Physical address: 14684 Value: 0
Virtual address: 24502 Physical address: 15030 Value: 23
Virtual address: 43446 Physical address: 15286 Value: 42
Virtual address: 20656 Physical address: 15536 Value: 0
Virtual address: 37795 Physical address: 15779 Value: -24
Virtual address: 27105 Physical address: 5601 Value: 0
Virtual address: 8464 Physical address: 15888 Value: 0
Virtual address: 36662 Physical address: 16182 Value: 35
Virtual address: 20196 Physical address: 16612 Value: 0
Virtual address: 25424 Physical address: 16720 Value: 0
Virtual address: 55854 Physical address: 16942 Value: 54
Virtual address: 52511 Physical address: 17183 Value: 71
Virtual address: 32850 Physical address: 17490 Value: 32
Virtual address: 1395 Physical address: 17779 Value: 92
Virtual address: 9151 Physical address: 3775 Value: -17
Virtual address: 29529 Physical address: 12377 Value: 0
Virtual address: 60218 Physical address: 17978 Value: 58
Virtual address: 34229 Physical address: 18357 Value: 0
Virtual address: 58605 Physical address: 1261 Value: 0
Virtual address: 56185 Physical address: 18553 Value: 0
Virtual address: 17461 Physical address: 18741 Value: 0
Virtual address: 38531 Physical address: 19075 Value: -96
Virtual address: 7208 Physical address: 19240 Value: 0
Virtual address: 57901 Physical address: 1837 Value: 0
Virtual address: 32578 Physical address: 19522 Value: 31
Virtual address: 38503 Physical address: 19047 Value: -103
Virtual address: 45993 Physical address: 8105 Value: 0
Virtual address: 63134 Physical address: 19870 Value: 61
Virtual address: 42511 Physical address: 19983 Value: -125
Virtual address: 30583 Physical address: 10359 Value: -35
Virtual address: 27915 Physical address: 20235 Value: 66
Virtual address: 61079 Physical address: 20631 Value: -91
Virtual address: 12790 Physical address: 20982 Value: 12
Virtual address: 35514 Physical address: 10682 Value: 34
Virtual address: 38805 Physical address: 21141 Value: 0
Virtual address: 13084 Physical address: 1564 Value: 0
Virtual address: 62712 Physical address: 21496 Value: 0
Virtual address: 46610 Physical address: 21522 Value: 45
Virtual address: 21460 Physical address: 21972 Value: 0
Virtual address: 41659 Physical address: 22203 Value: -82
Virtual address: 3650 Physical address: 22338 Value: 3
Virtual address: 31292 Physical address: 22588 Value: 0
Virtual address: 33515 Physical address: 23019 Value: -70
Virtual address: 54447 Physical address: 687 Value: 43
Virtual address: 63704 Physical address: 7384 Value: 0
Virtual address: 7517 Physical address: 23133 Value: 0
Virtual address: 45228 Physical address: 23468 Value: 0
Virtual address: 9962 Physical address: 23786 Value: 9
Virtual address: 55457 Physical address: 23969 Value: 0
Virtual address: 30459 Physical address: 5115 Value: -66
Virtual address: 42145 Physical address: 24225 Value: 0
Virtual address: 20666 Physical address: 15546 Value: 20
Virtual address: 32665 Physical address: 19609 Value: 0
Virtual address: 51536 Physical address: 11600 Value: 0
Virtual address: 3464 Physical address: 7560 Value: 0
Virtual address: 65398 Physical address: 24438 Value: 63
Virtual address: 17640 Physical address: 18920 Value: 0
Virtual address: 24550 Physical address: 15078 Value: 23
Virtual address: 29038 Physical address: 24686 Value: 28
Virtual address: 53534 Physical address: 6430 Value: 52
Virtual address: 40216 Physical address: 24856 Value: 0
Virtual address: 55950 Physical address: 17038 Value: 54
Virtual address: 31129 Physical address: 25241 Value: 0
Virtual address: 65216 Physical address: 8384 Value: 0
Virtual address: 1025 Physical address: 14593 Value: 0
Virtual address: 2066 Physical address: 25362 Value: 2
Virtual address: 7448 Physical address: 23064 Value: 0
Virtual address: 33158 Physical address: 25734 Value: 32
Virtual address: 10666 Physical address: 26026 Value: 10
Virtual address: 26126 Physical address: 26126 Value: 25
Virtual address: 39664 Physical address: 26608 Value: 0
Virtual address: 9690 Physical address: 26842 Value: 9
Virtual address: 22222 Physical address: 27086 Value: 21
Virtual address: 56962 Physical address: 5250 Value: 55
Virtual address: 39238 Physical address: 27206 Value: 38
Virtual address: 17160 Physical address: 27400 Value: 0
Virtual address: 40605 Physical address: 27805 Value: 0
Virtual address: 64471 Physical address: 28119 Value: -11
Virtual address: 12814 Physical address: 28174 Value: 12
Virtual address: 32017 Physical address: 28433 Value: 0
Virtual address: 46852 Physical address: 28676 Value: 0
Virtual address: 24720 Physical address: 4240 Value: 0
Virtual address: 10067 Physical address: 2643 Value: -44
Virtual address: 42123 Physical address: 24203 Value: 34
Virtual address: 28815 Physical address: 29071 Value: 35
Virtual address: 12774 Physical address: 20966 Value: 12
Virtual address: 22796 Physical address: 29196 Value: 0
Virtual address: 59342 Physical address: 29646 Value: 57
Virtual address: 31567 Physical address: 29775 Value: -45
Virtual address: 12538 Physical address: 30202 Value: 12
Virtual address: 55998 Physical address: 17086 Value: 54
Virtual address: 19246 Physical address: 30254 Value: 18
Virtual address: 51130 Physical address: 30650 Value: 49
Virtual address: 26637 Physical address: 30733 Value: 0
Virtual address: 50182 Physical address: 30982 Value: 49
Virtual address: 32916 Physical address: 17556 Value: 0
Virtual address: 42375 Physical address: 31367 Value: 97
Virtual address: 51178 Physical address: 30698 Value: 49
Virtual address: 58548 Physical address: 1204 Value: 0
Virtual address: 53216 Physical address: 31712 Value: 0
Virtual address: 36292 Physical address: 31940 Value: 0
Virtual address: 8697 Physical address: 16121 Value: 0
Virtual address: 35332 Physical address: 10500 Value: 0
Virtual address: 53490 Physical address: 32242 Value: 52
Virtual address: 21122 Physical address: 32386 Value: 20
Virtual address: 23579 Physical address: 32539 Value: 6
Virtual address: 43696 Physical address: 32944 Value: 0
Virtual address: 28172 Physical address: 33036 Value: 0
Virtual address: 11941 Physical address: 33445 Value: 0
Virtual address: 37120 Physical address: 11264 Value: 0
Virtual address: 26533 Physical address: 33701 Value: 0
Virtual address: 17080 Physical address: 33976 Value: 0
Virtual address: 45363 Physical address: 34099 Value: 76
Virtual address: 41245 Physical address: 12829 Value: 0
Virtual address: 22551 Physical address: 34327 Value: 5
Virtual address: 33812 Physical address: 34580 Value: 0
Virtual address: 26205 Physical address: 26205 Value: 0
Virtual address: 37585 Physical address: 35025 Value: 0
Virtual address: 17100 Physical address: 33996 Value: 0
Virtual address: 13254 Physical address: 1734 Value: 12
Virtual address: 7960 Physical address: 35096 Value: 0
Virtual address: 9835 Physical address: 23659 Value: -102
Virtual address: 36737 Physical address: 16257 Value: 0
Virtual address: 30312 Physical address: 4968 Value: 0
Virtual address: 8338 Physical address: 35474 Value: 8
Virtual address: 48254 Physical address: 35710 Value: 47
Virtual address: 31925 Physical address: 13749 Value: 0
Virtual address: 25224 Physical address: 9096 Value: 0
Virtual address: 65420 Physical address: 24460 Value: 0
Virtual address: 37150 Physical address: 11294 Value: 36
Virtual address: 2902 Physical address: 35926 Value: 2
Virtual address: 56455 Physical address: 36231 Value: 33
Virtual address: 45373 Physical address: 34109 Value: 0
Virtual address: 14207 Physical address: 36479 Value: -33
Virtual address: 26628 Physical address: 30724 Value: 0
Virtual address: 49260 Physical address: 36716 Value: 0
Virtual address: 20540 Physical address: 15420 Value: 0
Virtual address: 5773 Physical address: 13197 Value: 0
Virtual address: 10174 Physical address: 2750 Value: 9
Virtual address: 34812 Physical address: 37116 Value: 0
Virtual address: 13385 Physical address: 37193 Value: 0
Virtual address: 51138 Physical address: 30658 Value: 49
Virtual address: 4119 Physical address: 37399 Value: 5
Virtual address: 42495 Physical address: 31487 Value: 127
Virtual address: 61799 Physical address: 9575 Value: 89
Virtual address: 48009 Physical address: 37769 Value: 0
Virtual address: 42692 Physical address: 20164 Value: 0
Virtual address: 37522 Physical address: 34962 Value: 36
Virtual address: 22711 Physical address: 34487 Value: 45
Virtual address: 55945 Physical address: 17033 Value: 0
Virtual address: 37491 Physical address: 34931 Value: -100
Virtual address: 20098 Physical address: 16514 Value: 19
Virtual address: 37261 Physical address: 11405 Value: 0
Virtual address: 57343 Physical address: 38143 Value: -1
Virtual address: 63304 Physical address: 38216 Value: 0
Virtual address: 29748 Physical address: 38452 Value: 0
Virtual address: 21250 Physical address: 21762 Value: 20
Virtual address: 6363 Physical address: 38875 Value: 54
Virtual address: 17950 Physical address: 12062 Value: 17
Virtual address: 44596 Physical address: 38964 Value: 0
Virtual address: 10201 Physical address: 2777 Value: 0
Virtual address: 20869 Physical address: 39301 Value: 0
Virtual address: 28602 Physical address: 39610 Value: 27
Virtual address: 2363 Physical address: 39739 Value: 78
Virtual address: 49854 Physical address: 40126 Value: 48
Virtual address: 3286 Physical address: 40406 Value: 3
Virtual address: 45689 Physical address: 40569 Value: 0
Virtual address: 20076 Physical address: 16492 Value: 0
Virtual address: 33972 Physical address: 34740 Value: 0
Virtual address: 18272 Physical address: 40800 Value: 0
Virtual address: 12155 Physical address: 41083 Value: -34
Virtual address: 37231 Physical address: 11375 Value: 91
Virtual address: 6230 Physical address: 38742 Value: 6
Virtual address: 61913 Physical address: 9689 Value: 0
Virtual address: 3056 Physical address: 36080 Value: 0
Virtual address: 54112 Physical address: 41312 Value: 0
Virtual address: 37402 Physical address: 34842 Value: 36
Virtual address: 30997 Physical address: 25109 Value: 0
Virtual address: 19486 Physical address: 9758 Value: 19
Virtual address: 39918 Physical address: 41710 Value: 38
Virtual address: 35325 Physical address: 41981 Value: 0
Virtual address: 1946 Physical address: 42138 Value: 1
Virtual address: 43598 Physical address: 32846 Value: 42
Virtual address: 38576 Physical address: 19120 Value: 0
Virtual address: 53794 Physical address: 42274 Value: 52
Virtual address: 35223 Physical address: 41879 Value: 101
Virtual address: 4797 Physical address: 42685 Value: 0
Virtual address: 53304 Physical address: 32056 Value: 0
Virtual address: 51344 Physical address: 4752 Value: 0
Virtual address: 15562 Physical address: 42954 Value: 15
Virtual address: 35269 Physical address: 41925 Value: 0
Virtual address: 1948 Physical address: 42140 Value: 0
Virtual address: 33569 Physical address: 2849 Value: 0
Virtual address: 43532 Physical address: 32780 Value: 0
Virtual address: 48440 Physical address: 43064 Value: 0
Virtual address: 54041 Physical address: 41241 Value: 0
Virtual address: 8589 Physical address: 16013 Value: 0
Virtual address: 12289 Physical address: 29953 Value: 0
Virtual address: 37940 Physical address: 43316 Value: 0
Virtual address: 40239 Physical address: 24879 Value: 75
Virtual address: 20893 Physical address: 39325 Value: 0
Virtual address: 32530 Physical address: 19474 Value: 31
Virtual address: 9340 Physical address: 43644 Value: 0
Virtual address: 31760 Physical address: 13584 Value: 0
Virtual address: 27231 Physical address: 14431 Value: -105
Virtual address: 49635 Physical address: 44003 Value: 120
Virtual address: 20238 Physical address: 44046 Value: 19
Virtual address: 62102 Physical address: 44438 Value: 60
Virtual address: 59361 Physical address: 29665 Value: 0
Virtual address: 4500 Physical address: 4500 Value: 0
Virtual address: 44585 Physical address: 38953 Value: 0
Virtual address: 34790 Physical address: 37094 Value: 33
Virtual address: 14568 Physical address: 44776 Value: 0
Virtual address: 65533 Physical address: 24573 Value: 0
Virtual address: 16228 Physical address: 44900 Value: 0
Virtual address: 2911 Physical address: 35935 Value: -41
Virtual address: 28971 Physical address: 24619 Value: 74
Virtual address: 9115 Physical address: 3739 Value: -26
Virtual address: 14124 Physical address: 36396 Value: 0
Virtual address: 40378 Physical address: 25018 Value: 39
Virtual address: 33837 Physical address: 34605 Value: 0
Virtual address: 63804 Physical address: 45116 Value: 0
Virtual address: 61143 Physical address: 20695 Value: -75
Virtual address: 7949 Physical address: 35085 Value: 0
Virtual address: 21571 Physical address: 45379 Value: 16
Virtual address: 38237 Physical address: 6237 Value: 0
Virtual address: 34842 Physical address: 45594 Value: 34
Virtual address: 62455 Physical address: 46071 Value: -3
Virtual address: 32630 Physical address: 19574 Value: 31
Virtual address: 7462 Physical address: 23078 Value: 7
Virtual address: 12313 Physical address: 29977 Value: 0
Virtual address: 48799 Physical address: 46239 Value: -89
Virtual address: 3589 Physical address: 22277 Value: 0
Virtual address: 11345 Physical address: 46417 Value: 0
Virtual address: 30423 Physical address: 5079 Value: -75
Virtual address: 1724 Physical address: 46780 Value: 0
Virtual address: 30414 Physical address: 5070 Value: 29
Virtual address: 14088 Physical address: 36360 Value: 0
Virtual address: 37585 Physical address: 35025 Value: 0
Virtual address: 16688 Physical address: 46896 Value: 0
Virtual address: 26499 Physical address: 33667 Value: -32
Virtual address: 45345 Physical address: 34081 Value: 0
Virtual address: 49642 Physical address: 44010 Value: 48
Virtual address: 5667 Physical address: 13091 Value: -120
Virtual address: 31549 Physical address: 29757 Value: 0
Virtual address: 49002 Physical address: 47210 Value: 47
Virtual address: 50213 Physical address: 31013 Value: 0
Virtual address: 17022 Physical address: 33918 Value: 16
Virtual address: 47187 Physical address: 2387 Value: 20
Virtual address: 22458 Physical address: 47546 Value: 21
Virtual address: 15634 Physical address: 47634 Value: 15
Virtual address: 2937 Physical address: 35961 Value: 0
Virtual address: 38261 Physical address: 6261 Value: 0
Virtual address: 1677 Physical address: 46733 Value: 0
Virtual address: 13709 Physical address: 48013 Value: 0
Virtual address: 9850 Physical address: 23674 Value: 9
Virtual address: 59973 Physical address: 48197 Value: 0
Virtual address: 9709 Physical address: 26861 Value: 0
Virtual address: 65066 Physical address: 8234 Value: 63
Virtual address: 14321 Physical address: 36593 Value: 0
Virtual address: 45515 Physical address: 34251 Value: 114
Virtual address: 40539 Physical address: 27739 Value: -106
Virtual address: 51321 Physical address: 4729 Value: 0
Virtual address: 24271 Physical address: 48591 Value: -77
Virtual address: 16586 Physical address: 48842 Value: 16
Virtual address: 1515 Physical address: 17899 Value: 122
Virtual address: 45842 Physical address: 7954 Value: 44
Virtual address: 35860 Physical address: 48916 Value: 0
Virtual address: 2777 Physical address: 49369 Value: 0
Virtual address: 33929 Physical address: 34697 Value: 0
Virtual address: 2730 Physical address: 49322 Value: 2
Virtual address: 49847 Physical address: 40119 Value: -83
Virtual address: 60189 Physical address: 17949 Value: 0
Virtual address: 3690 Physical address: 22378 Value: 3
Virtual address: 57945 Physical address: 1881 Value: 0
Virtual address: 17593 Physical address: 18873 Value: 0
Virtual address: 53121 Physical address: 31617 Value: 0
Virtual address: 45931 Physical address: 8043 Value: -38
Virtual address: 22286 Physical address: 47374 Value: 21
Virtual address: 43815 Physical address: 49447 Value: -55
Virtual address: 56357 Physical address: 36133 Value: 0
Virtual address: 4902 Physical address: 49702 Value: 4
Virtual address: 35720 Physical address: 50056 Value: 0
Virtual address: 3608 Physical address: 22296 Value: 0
Virtual address: 1623 Physical address: 46679 Value: -107
Virtual address: 5410 Physical address: 50210 Value: 5
Virtual address: 24768 Physical address: 4288 Value: 0
Virtual address: 39276 Physical address: 27244 Value: 0
Virtual address: 56133 Physical address: 18501 Value: 0
Virtual address: 45270 Physical address: 23510 Value: 44
Virtual address: 28524 Physical address: 39532 Value: 0
Virtual address: 33479 Physical address: 22983 Value: -79
Virtual address: 40266 Physical address: 24906 Value: 39
Virtual address: 42970 Physical address: 11994 Value: 41
Virtual address: 20307 Physical address: 44115 Value: -44
Virtual address: 37380 Physical address: 34820 Value: 0
Virtual address: 37575 Physical address: 35015 Value: -79
Virtual address: 3261 Physical address: 40381 Value: 0
Virtual address: 7061 Physical address: 50581 Value: 0
Virtual address: 61331 Physical address: 50835 Value: -28
Virtual address: 58575 Physical address: 1231 Value: 51
Virtual address: 24540 Physical address: 15068 Value: 0
Virtual address: 13678 Physical address: 47982 Value: 13
Virtual address: 7514 Physical address: 23130 Value: 7
Virtual address: 37803 Physical address: 15787 Value: -22
Virtual address: 474 Physical address: 10202 Value: 0
Virtual address: 18424 Physical address: 40952 Value: 0
Virtual address: 31203 Physical address: 25315 Value: 120
Virtual address: 34282 Physical address: 18410 Value: 33
Virtual address: 48653 Physical address: 46093 Value: 0
Virtual address: 33652 Physical address: 2932 Value: 0
Virtual address: 31362 Physical address: 22658 Value: 30
Virtual address: 8396 Physical address: 35532 Value: 0
Virtual address: 53853 Physical address: 42333 Value: 0
Virtual address: 36144 Physical address: 31792 Value: 0
Virtual address: 40181 Physical address: 6133 Value: 0
Virtual address: 6563 Physical address: 51107 Value: 104
Virtual address: 46955 Physical address: 28779 Value: -38
Virtual address: 3190 Physical address: 40310 Value: 3
Virtual address: 63429 Physical address: 38341 Value: 0
Virtual address: 43902 Physical address: 49534 Value: 42
Virtual address: 17077 Physical address: 33973 Value: 0
Virtual address: 32287 Physical address: 51231 Value: -121
Virtual address: 47452 Physical address: 51548 Value: 0
Virtual address: 9096 Physical address: 3720 Value: 0
Virtual address: 16419 Physical address: 48675 Value: 8
Virtual address: 16642 Physical address: 46850 Value: 16
Virtual address: 14913 Physical address: 51777 Value: 0
Virtual address: 14025 Physical address: 52169 Value: 0
Virtual address: 59322 Physical address: 29626 Value: 57
Virtual address: 46836 Physical address: 21748 Value: 0
Virtual address: 38333 Physical address: 6333 Value: 0
Virtual address: 53489 Physical address: 32241 Value: 0
Virtual address: 58695 Physical address: 52295 Value: 81
Virtual address: 46559 Physical address: 52703 Value: 119
Virtual address: 43942 Physical address: 49574 Value: 42
Virtual address: 25515 Physical address: 16811 Value: -22
Virtual address: 26642 Physical address: 30738 Value: 26
Virtual address: 2657 Physical address: 49249 Value: 0
Virtual address: 1366 Physical address: 17750 Value: 1
Virtual address: 42776 Physical address: 11800 Value: 0
Virtual address: 11137 Physical address: 7041 Value: 0
Virtual address: 38636 Physical address: 19180 Value: 0
Virtual address: 12242 Physical address: 41170 Value: 11
Virtual address: 40340 Physical address: 24980 Value: 0
Virtual address: 9120 Physical address: 3744 Value: 0
Virtual address: 35684 Physical address: 50020 Value: 0
Virtual address: 4679 Physical address: 42567 Value: -111
Virtual address: 45877 Physical address: 7989 Value: 0
Virtual address: 36820 Physical address: 16340 Value: 0
Virtual address: 28564 Physical address: 39572 Value: 0
Virtual address: 49308 Physical address: 36764 Value: 0
Virtual address: 10996 Physical address: 52980 Value: 0
Virtual address: 27755 Physical address: 53099 Value: 26
Virtual address: 22947 Physical address: 29347 Value: 104
Virtual address: 53631 Physical address: 6527 Value: 95
Virtual address: 1378 Physical address: 17762 Value: 1
Virtual address: 24954 Physical address: 53370 Value: 24
Virtual address: 53255 Physical address: 32007 Value: 1
Virtual address: 4886 Physical address: 49686 Value: 4
Virtual address: 25955 Physical address: 53603 Value: 88
Virtual address: 45677 Physical address: 40557 Value: 0
Virtual address: 7017 Physical address: 50537 Value: 0
Virtual address: 59994 Physical address: 48218 Value: 58
Virtual address: 39697 Physical address: 41489 Value: 0
Virtual address: 50311 Physical address: 31111 Value: 33
Virtual address: 62878 Physical address: 8606 Value: 61
Virtual address: 13172 Physical address: 1652 Value: 0
Virtual address: 3029 Physical address: 36053 Value: 0
Virtual address: 54655 Physical address: 5759 Value: 95
Virtual address: 30551 Physical address: 10327 Value: -43
Virtual address: 894 Physical address: 53886 Value: 0
Virtual address: 62970 Physical address: 8698 Value: 61
Virtual address: 9476 Physical address: 26628 Value: 0
Virtual address: 18292 Physical address: 40820 Value: 0
Virtual address: 39207 Physical address: 27175 Value: 73
Virtual address: 60095 Physical address: 48319 Value: -81
Virtual address: 54265 Physical address: 41465 Value: 0
Virtual address: 36385 Physical address: 54049 Value: 0
Virtual address: 11268 Physical address: 46340 Value: 0
Virtual address: 38633 Physical address: 19177 Value: 0
Virtual address: 53801 Physical address: 42281 Value: 0
Virtual address: 61112 Physical address: 20664 Value: 0
Virtual address: 18432 Physical address: 54272 Value: 0
Virtual address: 36966 Physical address: 54630 Value: 36
Virtual address: 1506 Physical address: 17890 Value: 1
Virtual address: 52877 Physical address: 54925 Value: 0
Virtual address: 11643 Physical address: 55163 Value: 94
Virtual address: 34963 Physical address: 45715 Value: 36
Virtual address: 15806 Physical address: 47806 Value: 15
Virtual address: 56874 Physical address: 5162 Value: 55
Virtual address: 4357 Physical address: 4357 Value: 0
Virtual address: 41851 Physical address: 55419 Value: -34
Virtual address: 24349 Physical address: 14877 Value: 0
Virtual address: 4737 Physical address: 42625 Value: 0
Virtual address: 52113 Physical address: 55697 Value: 0
Virtual address: 36027 Physical address: 49083 Value: 46
Virtual address: 26057 Physical address: 53705 Value: 0
Virtual address: 60796 Physical address: 380 Value: 0
Virtual address: 501 Physical address: 10229 Value: 0
Virtual address: 57391 Physical address: 55855 Value: 11
Virtual address: 63946 Physical address: 45258 Value: 62
Virtual address: 63885 Physical address: 45197 Value: 0
Virtual address: 54760 Physical address: 5864 Value: 0
Virtual address: 6248 Physical address: 38760 Value: 0
Virtual address: 52205 Physical address: 55789 Value: 0
Virtual address: 765 Physical address: 56317 Value: 0
Virtual address: 12709 Physical address: 20901 Value: 0
Virtual address: 59880 Physical address: 56552 Value: 0
Virtual address: 12763 Physical address: 20955 Value: 118
Virtual address: 26349 Physical address: 26349 Value: 0
Virtual address: 23698 Physical address: 32658 Value: 23
Virtual address: 56625 Physical address: 56625 Value: 0
Virtual address: 19147 Physical address: 57035 Value: -78
Virtual address: 4477 Physical address: 4477 Value: 0
Virtual address: 58705 Physical address: 52305 Value: 0
Virtual address: 50021 Physical address: 57189 Value: 0
Virtual address: 21218 Physical address: 32482 Value: 20
Virtual address: 26976 Physical address: 5472 Value: 0
Virtual address: 30394 Physical address: 5050 Value: 29
Virtual address: 21437 Physical address: 21949 Value: 0
Virtual address: 20532 Physical address: 15412 Value: 0
Virtual address: 26165 Physical address: 26165 Value: 0
Virtual address: 55977 Physical address: 17065 Value: 0
Virtual address: 52385 Physical address: 57505 Value: 0
Virtual address: 18541 Physical address: 54381 Value: 0
Virtual address: 56204 Physical address: 18572 Value: 0
Virtual address: 56530 Physical address: 36306 Value: 55
Virtual address: 3489 Physical address: 7585 Value: 0
Virtual address: 43347 Physical address: 15187 Value: 84
Virtual address: 889 Physical address: 53881 Value: 0
Virtual address: 14885 Physical address: 51749 Value: 0
Virtual address: 11186 Physical address: 7090 Value: 10
Virtual address: 23775 Physical address: 32735 Value: 55
Virtual address: 51889 Physical address: 11185 Value: 0
Number of Translated Addresses = 500
Page Faults = 225
Page Fault Rate = 0.450
TLB Hits = 27
TLB Hit Rate = 0.054
